grpc/src/main/java/com/linecorp/armeria/internal/common/grpc/HttpStreamReader.java

```diff
@@ -166,12 +166,12 @@ public void onNext(HttpObject obj) {
 
     @Override
     public void onError(Throwable cause) {
-        // Handled by accept() below.
+        // Handled by apply() below.
```

Comment:
Can we handle this without using `BiFunction`?

Reply:
I've tried, but some tests failed. 😅
Let me fix this another PR.